{
  "uid" : "d10af88de7a0b588",
  "name" : "Verify redirection to used resources page not authorized user.",
  "fullName" : "tests.main_page_test.TestMainPage#test_mp_08_verify_redirection_to_used_resources_page",
  "historyId" : "43df35c0ef3ab6719310f8b9b9ee7530",
  "time" : {
    "start" : 1739584599254,
    "stop" : 1739584639689,
    "duration" : 40435
  },
  "status" : "broken",
  "statusMessage" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources",
  "statusTrace" : "self = <pages.main_page.MainPage object at 0x7f3c0d9b2960>\nurl = 'https://brainup.site/used-resources'\n\n    def check_expected_link(self, url):\n        with allure.step(f'Check url is present: {url}'):\n            try:\n>               return Wait(self.driver, self.timeout).until(\n                    ec.url_to_be(url), message=f\"Can't find element by locator {url}\")\n\npages/base_page.py:68: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"934e25a6375aab180bf38b64e99382f7\")>\nmethod = <function url_to_be.<locals>._predicate at 0x7f3c0d97d440>\nmessage = \"Can't find element by locator https://brainup.site/used-resources\"\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources\n\n/opt/hostedtoolcache/Python/3.12.9/x64/lib/python3.12/site-packages/selenium/webdriver/support/wait.py:95: TimeoutException\n\nDuring handling of the above exception, another exception occurred:\n\nself = <tests.main_page_test.TestMainPage object at 0x7f3c0db60890>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"934e25a6375aab180bf38b64e99382f7\")>\nmain_page_open = None\n\n    @allure.title(\"Verify redirection to used resources page not authorized user.\")\n    def test_mp_08_verify_redirection_to_used_resources_page(self, driver, main_page_open):\n        page = MainPage(driver)\n>       resources_page_url = page.open_used_resources_page()\n\ntests/main_page_test.py:57: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/main_page.py:68: in open_used_resources_page\n    self.check_expected_link(MainPageLinks.URL_USED_RESOURCES_PAGE)\npages/base_page.py:72: in check_expected_link\n    return Wait(self.driver, self.timeout).until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"934e25a6375aab180bf38b64e99382f7\")>\nmethod = <function url_to_be.<locals>._predicate at 0x7f3c0d97f240>\nmessage = \"Can't find element by locator https://brainup.site/used-resources\"\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources\n\n/opt/hostedtoolcache/Python/3.12.9/x64/lib/python3.12/site-packages/selenium/webdriver/support/wait.py:95: TimeoutException",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "driver",
    "time" : {
      "start" : 1739584595110,
      "stop" : 1739584595938,
      "duration" : 828
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "hasContent" : false,
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false
  }, {
    "name" : "main_page_open",
    "time" : {
      "start" : 1739584595939,
      "stop" : 1739584599254,
      "duration" : 3315
    },
    "status" : "passed",
    "steps" : [ {
      "name" : "Open page: https://brainup.site/",
      "time" : {
        "start" : 1739584595939,
        "stop" : 1739584599254,
        "duration" : 3315
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ {
        "name" : "driver",
        "value" : "<selenium.webdriver.chrome.webdriver.WebDriver (session=\"934e25a6375aab180bf38b64e99382f7\")>"
      } ],
      "hasContent" : true,
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "hasContent" : true,
    "stepsCount" : 1,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false
  }, {
    "name" : "clear_allure_results_folder",
    "time" : {
      "start" : 1739583657297,
      "stop" : 1739583657298,
      "duration" : 1
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "hasContent" : false,
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false
  } ],
  "testStage" : {
    "status" : "broken",
    "statusMessage" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources",
    "statusTrace" : "self = <pages.main_page.MainPage object at 0x7f3c0d9b2960>\nurl = 'https://brainup.site/used-resources'\n\n    def check_expected_link(self, url):\n        with allure.step(f'Check url is present: {url}'):\n            try:\n>               return Wait(self.driver, self.timeout).until(\n                    ec.url_to_be(url), message=f\"Can't find element by locator {url}\")\n\npages/base_page.py:68: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"934e25a6375aab180bf38b64e99382f7\")>\nmethod = <function url_to_be.<locals>._predicate at 0x7f3c0d97d440>\nmessage = \"Can't find element by locator https://brainup.site/used-resources\"\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources\n\n/opt/hostedtoolcache/Python/3.12.9/x64/lib/python3.12/site-packages/selenium/webdriver/support/wait.py:95: TimeoutException\n\nDuring handling of the above exception, another exception occurred:\n\nself = <tests.main_page_test.TestMainPage object at 0x7f3c0db60890>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"934e25a6375aab180bf38b64e99382f7\")>\nmain_page_open = None\n\n    @allure.title(\"Verify redirection to used resources page not authorized user.\")\n    def test_mp_08_verify_redirection_to_used_resources_page(self, driver, main_page_open):\n        page = MainPage(driver)\n>       resources_page_url = page.open_used_resources_page()\n\ntests/main_page_test.py:57: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/main_page.py:68: in open_used_resources_page\n    self.check_expected_link(MainPageLinks.URL_USED_RESOURCES_PAGE)\npages/base_page.py:72: in check_expected_link\n    return Wait(self.driver, self.timeout).until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"934e25a6375aab180bf38b64e99382f7\")>\nmethod = <function url_to_be.<locals>._predicate at 0x7f3c0d97f240>\nmessage = \"Can't find element by locator https://brainup.site/used-resources\"\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources\n\n/opt/hostedtoolcache/Python/3.12.9/x64/lib/python3.12/site-packages/selenium/webdriver/support/wait.py:95: TimeoutException",
    "steps" : [ {
      "name" : "Open used resources page",
      "time" : {
        "start" : 1739584599254,
        "stop" : 1739584639688,
        "duration" : 40434
      },
      "status" : "broken",
      "statusMessage" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources\n\n",
      "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.9/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/pages/main_page.py\", line 68, in open_used_resources_page\n    self.check_expected_link(MainPageLinks.URL_USED_RESOURCES_PAGE)\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/pages/base_page.py\", line 72, in check_expected_link\n    return Wait(self.driver, self.timeout).until(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.9/x64/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 95, in until\n    raise TimeoutException(message, screen, stacktrace)\n",
      "steps" : [ {
        "name" : "Check url is present: https://brainup.site/used-resources",
        "time" : {
          "start" : 1739584599424,
          "stop" : 1739584639688,
          "duration" : 40264
        },
        "status" : "broken",
        "statusMessage" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources\n\n",
        "statusTrace" : "  File \"/home/runner/work/auto-tests-python/auto-tests-python/pages/base_page.py\", line 72, in check_expected_link\n    return Wait(self.driver, self.timeout).until(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.9/x64/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 95, in until\n    raise TimeoutException(message, screen, stacktrace)\n",
        "steps" : [ ],
        "attachments" : [ ],
        "parameters" : [ ],
        "hasContent" : true,
        "stepsCount" : 0,
        "attachmentsCount" : 0,
        "shouldDisplayMessage" : true
      } ],
      "attachments" : [ ],
      "parameters" : [ ],
      "hasContent" : true,
      "stepsCount" : 1,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false
    } ],
    "attachments" : [ {
      "uid" : "a450d90482e4da31",
      "name" : "Screenshot 2025-02-15 01:57:19.790610",
      "source" : "a450d90482e4da31.png",
      "type" : "image/png",
      "size" : 61086
    } ],
    "parameters" : [ ],
    "hasContent" : true,
    "stepsCount" : 2,
    "attachmentsCount" : 1,
    "shouldDisplayMessage" : true
  },
  "afterStages" : [ {
    "name" : "driver::0",
    "time" : {
      "start" : 1739584639792,
      "stop" : 1739584639857,
      "duration" : 65
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "hasContent" : false,
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false
  } ],
  "labels" : [ {
    "name" : "epic",
    "value" : "Main Page."
  }, {
    "name" : "parentSuite",
    "value" : "tests"
  }, {
    "name" : "suite",
    "value" : "main_page_test"
  }, {
    "name" : "subSuite",
    "value" : "TestMainPage"
  }, {
    "name" : "host",
    "value" : "fv-az1307-937"
  }, {
    "name" : "thread",
    "value" : "2413-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.main_page_test"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 53,
        "skipped" : 0,
        "passed" : 78,
        "unknown" : 0,
        "total" : 131
      },
      "items" : [ {
        "uid" : "8ac1dfb645d737e4",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/185//#testresult/8ac1dfb645d737e4",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources",
        "time" : {
          "start" : 1739498341968,
          "stop" : 1739498382449,
          "duration" : 40481
        }
      }, {
        "uid" : "82a98459e2a99bbd",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/184//#testresult/82a98459e2a99bbd",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources",
        "time" : {
          "start" : 1739411926537,
          "stop" : 1739411966955,
          "duration" : 40418
        }
      }, {
        "uid" : "51a0e39961bab248",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/183//#testresult/51a0e39961bab248",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources",
        "time" : {
          "start" : 1739325549337,
          "stop" : 1739325589781,
          "duration" : 40444
        }
      }, {
        "uid" : "f41c10f938297557",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/182//#testresult/f41c10f938297557",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources",
        "time" : {
          "start" : 1739239061601,
          "stop" : 1739239102002,
          "duration" : 40401
        }
      }, {
        "uid" : "95325dff87a2968",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/181//#testresult/95325dff87a2968",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources",
        "time" : {
          "start" : 1739152863347,
          "stop" : 1739152903844,
          "duration" : 40497
        }
      }, {
        "uid" : "4832c7d00382bd1c",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/180//#testresult/4832c7d00382bd1c",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources",
        "time" : {
          "start" : 1739066516096,
          "stop" : 1739066556533,
          "duration" : 40437
        }
      }, {
        "uid" : "830e3f2d38ee8a8b",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/179//#testresult/830e3f2d38ee8a8b",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources",
        "time" : {
          "start" : 1738979662260,
          "stop" : 1738979702706,
          "duration" : 40446
        }
      }, {
        "uid" : "1f1178244909a5f5",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/178//#testresult/1f1178244909a5f5",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources",
        "time" : {
          "start" : 1738893460853,
          "stop" : 1738893501320,
          "duration" : 40467
        }
      }, {
        "uid" : "a30fd02108900d77",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/177//#testresult/a30fd02108900d77",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources",
        "time" : {
          "start" : 1738807052521,
          "stop" : 1738807092993,
          "duration" : 40472
        }
      }, {
        "uid" : "900f2be626eef4df",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/176//#testresult/900f2be626eef4df",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources",
        "time" : {
          "start" : 1738720748269,
          "stop" : 1738720788702,
          "duration" : 40433
        }
      }, {
        "uid" : "7023ff7bb96d53cc",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/175//#testresult/7023ff7bb96d53cc",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources",
        "time" : {
          "start" : 1738634180056,
          "stop" : 1738634220545,
          "duration" : 40489
        }
      }, {
        "uid" : "efce4b66274edd7d",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/174//#testresult/efce4b66274edd7d",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources",
        "time" : {
          "start" : 1738547948621,
          "stop" : 1738547989083,
          "duration" : 40462
        }
      }, {
        "uid" : "5ecc4c19968dacc8",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/173//#testresult/5ecc4c19968dacc8",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources",
        "time" : {
          "start" : 1738461616816,
          "stop" : 1738461657275,
          "duration" : 40459
        }
      }, {
        "uid" : "ce08f0e839a2ab8c",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/172//#testresult/ce08f0e839a2ab8c",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources",
        "time" : {
          "start" : 1738375347420,
          "stop" : 1738375387928,
          "duration" : 40508
        }
      }, {
        "uid" : "a9928a542c33a82f",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/170//#testresult/a9928a542c33a82f",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources",
        "time" : {
          "start" : 1738202125904,
          "stop" : 1738202166336,
          "duration" : 40432
        }
      }, {
        "uid" : "72cf5707e7968da6",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/169//#testresult/72cf5707e7968da6",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources",
        "time" : {
          "start" : 1738115723613,
          "stop" : 1738115764050,
          "duration" : 40437
        }
      }, {
        "uid" : "c7efec0d4fcde38b",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/168//#testresult/c7efec0d4fcde38b",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources",
        "time" : {
          "start" : 1738029413444,
          "stop" : 1738029453868,
          "duration" : 40424
        }
      }, {
        "uid" : "9cc7859b90df7af7",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/167//#testresult/9cc7859b90df7af7",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources",
        "time" : {
          "start" : 1737943039169,
          "stop" : 1737943079624,
          "duration" : 40455
        }
      }, {
        "uid" : "bfe682ff3ff5f42d",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/166//#testresult/bfe682ff3ff5f42d",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources",
        "time" : {
          "start" : 1737856686196,
          "stop" : 1737856726724,
          "duration" : 40528
        }
      }, {
        "uid" : "c4eeaeae961dc349",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/165//#testresult/c4eeaeae961dc349",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/used-resources",
        "time" : {
          "start" : 1737769926835,
          "stop" : 1737769967270,
          "duration" : 40435
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "d10af88de7a0b588.json",
  "parameterValues" : [ ]
}